integration-tests/amazon-lambda-http-resteasy/src/test/java/io/quarkus/it/amazon/lambda/AmazonLambdaSimpleTestCase.java

```diff
@@ -105,4 +107,19 @@ public void testPostEmpty() throws Exception {
 
     }
 
+    @Test
+    public void testAwsProxyRequestContext() throws Exception {
+        ApiGatewayAuthorizerContext authorizerContext = new ApiGatewayAuthorizerContext();
+        authorizerContext.setContextValue("authorizerValue", "testValue");
+        AwsProxyRequestContext awsProxyRequestContext = new AwsProxyRequestContext();
```

Comment:
Your test is incomplete.  You should have a jaxrs resource or something that injects the context object with @Context